File: Задача 1/normFunc.js
function norm(maxN){

    let resultArray = [];
    // перебираем все простые числа, пока массив с результатом не достигнет заданного
    for (let n = 2; resultArray.length < maxN; n++) {
        // вводим вспомгательный маркер (мне так проще)
        let check = false;  
        // Проверка чисел на деление
        for (let resultArrayIndex = 0; resultArrayIndex < resultArray.length; resultArrayIndex++) {

            if (n % resultArray[resultArrayIndex] === 0 && !(n / resultArray[resultArrayIndex] === 1)) {
                check = true
                break
            }
            
        }

        if (check === false) {
            resultArray.push(n)
        };

    }

    return resultArray;

}


console.time();
console.log(norm(process.argv[2]));
console.timeEnd();


// Math.sqrt(n) < resultArray[arrIndex]

File: Задача 1/norm_func.py
from datetime import datetime
import time


def norm(maxN):
    start_time = datetime.now()

    result_list = [];
    n = 2

    while len(result_list) < maxN:

        check = False;  
        result_array_numb = 0

        for result_array_numb in result_list:

            if n % result_array_numb == 0 & (n / result_array_numb != 1):
                check = True
                break


        if (check == False):
            result_list.append(n)

        n = n + 1
        
    print(datetime.now() - start_time)
    return result_list;



if __name__ == "__main__":
    (norm(100000))